.devcontainer/devcontainer.json

```diff
@@ -38,6 +47,50 @@
       "!include_dir_list scalar",
       "!include_dir_merge_list scalar",
       "!include_dir_merge_named scalar"
+    ],
+    // python common
+    "[python]": {
+      "editor.defaultFormatter": "ms-python.python",
+      "editor.codeActionsOnSave": {
+        "source.organizeImports": true
+      }
+    },
+    "python.linting.enabled": true,
+    "python.pythonPath": "/usr/local/bin/python",
+    // python linting
+    "python.linting.banditEnabled": true,
+    "python.linting.banditArgs": [
+      "--format=custom",
+      "--configfile=tests/bandit.yaml"
+    ],
+    "python.linting.flake8Enabled": true,
+    "python.linting.flake8Args": ["--config=./setup.cfg"],
+    "python.linting.mypyEnabled": true,
+    "python.linting.mypyArgs": [
+      "--follow-imports=silent",
+      "--ignore-missing-imports",
+      "--show-column-numbers",
+      "--no-pretty",
+      "--config-file=./mypy.ini"
+    ],
+    "python.linting.pylintEnabled": false,
+    "python.linting.pylamaEnabled": false,
+    "python.formatting.provider": "black",
+    "python.formatting.blackArgs": [
+      "--config=./pyproject.toml",
+      "--line-length=88"
+    ],
+    "isort.args": ["--settings-path=./pyproject.toml"],
+    "pylint.args": ["--rcfile=./pyproject.toml"],
+    // python tests
+    "python.testing.promptToConfigure": false,
+    // disabled by default due to extra needed requirements
+    "python.testing.pytestEnabled": false,
+    "python.testing.unittestEnabled": false,
+    "python.testing.pytestArgs": [
+      "--color=no"
+      // limit pytest to just a single component
+      // "--override-ini=testpaths=tests/components/yourcomponent"
```

Comment:
I think this doesn't belong here? If we want to help out on this one specific, we need to ask it when running a task.

Reply:
Just an example to show the user how to do it. You cannot attach the debugger when it is a task.

Reply:
It should not be resolved like this, let's create a task instead (which asks for input on which integration to test).

Reply:
That goes back to you cannot debug tasks. You have to use a debug launch task which is done through the test explorer. 

There is already the Code Coverage task that handles what you are talking about really well. But there is no way to debug tests for a specific component without enabling tests for everything.

Reply:
You are core a dev, so you generally work on everything, but a lot of contributors do not. They tend only work on 1 or a small handful of integrations. This commented out to show it as an example for a reason. It is for someone who only work on a small number of components that goes "man, I do not want to wait 2 minutes to reload tests just so I can debug my new test".